### PR TITLE
Create change_group.yaml

### PR DESCRIPTION
**Pull request by daviedanielle**
OS policy assignment that sets the /var/log/auth.log group to logaccess.

### CODE REVIEW

File: examples/OSPolicyAssignments/gcloud/change_group.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+             validate:
+               # Checks if the group is logaccess. If yes, exits  with code 100. If no,
+                # exits with code 101 and proceeds to the `enforce` step.
+               script:
+                 if stat -c '%G /var/log/auth.log | grep -q 'logaccess'; then exit 100; else exit 101; fi
+               interpreter: SHELL
+             enforce:
+               # Changes the group to logaccess and exits with code 100.
+               script:
+                 sudo chgrp logaccess /var/log/auth.log && exit 100
```
Comment (adjackura) [inline examples/OSPolicyAssignments/gcloud/change_group.yaml:18]: drop sudo here

Comment (daviedanielle) [reply examples/OSPolicyAssignments/gcloud/change_group.yaml]: Done

**Review by adjackura — APPROVED** [review]

**Pull request merged**